tests/load/gcp/lma_startup_script.tpl.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+set -euxo pipefail
+
+systemctl disable systemd-tmpfiles-clean.timer --now
+microk8s status --wait-ready
+microk8s.enable storage dns ingress
+
+IPADDR=$(ip -4 route | grep default | grep -oP 'src \K\S+')
```

Comment:
Is this trying to get the private IP of the instance? Either way, using the json output of the `ip` command might be friendlier

```suggestion
IPADDR=$(ip -4 -j route | jq -r '.[] | select(.dst | contains("default")) | .src')
```